packages/components/Menu/src/MenuList/MenuList.types.ts

```diff
@@ -12,6 +12,7 @@ export interface MenuListTokens extends LayoutTokens, IBackgroundColorTokens {
    * Space between items in pixels
    */
   gap?: number;
+  hasMaxHeight?: MenuListTokens;
```

Comment:
Hmm, was it intentional to have it as a token?

Reply:
Yes, in order to support this as a state for styling, the type should be a token